Resources/Prototypes/DeltaV/Catalog/Cargo/cargo_vending.yml

```diff
@@ -4,6 +4,6 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockPrideFilled
-  cost: 1000
+  cost: 1250
```

Comment:
This should be mentioned in your Changelog too.

Reply:
i Believe i have added this to the change log now but correct me if i did it poorly